src/mudlet.h

```diff
@@ -401,6 +408,7 @@ class mudlet : public QMainWindow, public Ui::main_window
     // How many graphemes do we need before we run the spell checker on a "word" in the command line:
     int mMinLengthForSpellCheck = 3;
     bool mDrawUpperLowerLevels = true;
+    bool mShowTabConnectionIndicators = true; // Global preference for showing connection status indicators on tabs
```

Comment:
I'd prefer this to be off by default for a cleaner look'n feel to the client:

```suggestion
    bool mShowTabConnectionIndicators = false; // Global preference for showing connection status indicators on tabs
```